pkg/controllermanager/apis/config/types.go

```diff
@@ -12,10 +12,12 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package componentconfig
+package config
 
 import (
+	apimachineryconfig "k8s.io/apimachinery/pkg/apis/config"
```

Comment:
Can you add a comment that in the future this should be 

`k8s.io/component-base/config`

See https://github.com/kubernetes/component-base/blob/7b44618cd0055577ef46bce14870b1bee6656ed9/config/v1alpha1/types.go